DataSetExplorer/DataSetSerializer/DataSetWithMetricsExporter.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using DataSetExplorer.DataSetBuilder.Model;
+using OfficeOpenXml;
+using OfficeOpenXml.Style;
+using RepositoryCompiler.CodeModel.CaDETModel.CodeItems;
+
+namespace DataSetExplorer.DataSetSerializer
+{
+    class DataSetWithMetricsExporter
+    {
+        private readonly string _templatePath = "../../../DataSetSerializer/Template/Dataset_Metrics_Template.xlsx";
+        private readonly string _exportPath;
+        private ExcelPackage template;
```

Comment:
private members start with _. Also "template" is a misleading name.

Reply:
Is the _excelFile ok?